tool/update_copyright.rb

```diff
@@ -51,18 +51,23 @@
 truffle_paths = %w[
   src
   test/truffle
-  spec/
+  spec/truffle
 ] + [__FILE__]
 
 excludes = %w[
   test/truffle/pack-real-usage.rb
   test/truffle/cexts
+  test/truffle/ecosystem
   src/main/c/openssl
 ]
 
 # Until those all have copyright headers
 excludes << "src/main/java/org/truffleruby/parser"
```

Comment:
@chrisseaton  I'd like to remove this line.
But for that, all parser files should have a copyright header.
Could you add them?
The list of files with missing copyrights is https://gist.github.com/eregon/561361e30536d2402544778a67af4eaa